global_frame: "map"

image_downsample: 4
optical_flow_win_size: 31

edge_threshold: 62
patch_size: 62

bucket_size: 25
features_per_bucket: 1
